scss/6-components/_sidebar.scss

```diff
@@ -0,0 +1,46 @@
+.cads-sidebar {
+  padding-left: $cads-spacing-4;
+
+  @include cads-media-breakpoint-only(sm) {
+    padding-left: 0;
+  }
+
+  &__title {
+    color: $cads-language__secondary-text-colour;
+  }
+
+  &__section-title {
+    color: $cads-language__secondary-text-colour;
+
+    &--small {
```

Comment:
Can I use a modifier here? public-website has a separate class

Reply:
Is there another variant? If not you can just put it all into the same selector ruleset.

Reply:
Ahhh I just had a look in public website - I think this has come from the fact the `AdviceCollectionLinks` component is also used to render the advice list pages.  We'll probably change that when this component is done - the sidebar shouldn't concern itself with the styles/markup for the advice collection lists pages.  So we can just have one ruleset here.